roles/gi-setup/tasks/gi-install.yml

```diff
@@ -170,7 +165,7 @@
 
 - name: gi-install | Set installer command
   set_fact:
-    installer_command: "{{ grid_home }}/gridSetup.sh -silent -responseFile {{ swlib_unzip_path }}/grid_install.rsp {{ rel_patch|default('') }} {{ mgmt_option|default('') }} -ignorePrereqFailure"
+    installer_command: "{{ installer_command }} -silent -responseFile {{ swlib_unzip_path }}/grid_install.rsp {{ rel_patch|default('') }} {{ mgmt_option|default('') }}"
```

Comment:
Nit: If it were me I would list prereqOption here, just so that all the arguments for the installer command are listed.

Reply:
it is listed through information task, for example:
....
TASK [gi-setup : gi-install | Information] ***************************************************************************************************************************************************************************************************************
ok: [oel75-si2-2] => {
    "msg": "Using installer cmd: /u01/app/19.3.0/grid/gridSetup.sh -ignorePrereqFailure -silent -responseFile /u01/oracle_install/grid_install.rsp -applyRU /u01/oracle_install/30783556/30899722 -J-Doracle.install.mgmtDB=false -J-Doracle.install.mgmtDB.CDB=false -J-Doracle.install.crs.enableRemoteGIMR=false"
}

....

TASK [rdbms-setup : rdbms-install | Information] *********************************************************************************************************************************************************************************************************
ok: [oel75-si2-2] => {
    "msg": "Using installer cmd: /u01/app/oracle/product/19.3.0/dbhome_1/runInstaller -silent -waitforcompletion -responseFile /u01/oracle_install/db_install.rsp -applyRU /u01/oracle_install/30783556/30899722 -applyOneOffs /u01/oracle_install/30783556/30805684 -ignorePrereqFailure"
}